### DISCUSSION TITLE
Keep selected dates in focus when clicking away

### DISCUSSION THREAD

**Discussion by xsrokhan** [wojtekmaj/react-calendar#874] [Q&A] [answered]
Currently, the dates get :focus styles applied only when in focus, but as soon as the user clicks away, the selected date loses focus and it's not clear which date was selected. Is there a way to persist the :focus styles on selected dates? Here's an example, click on the New button to have the calendar displayed https://stackblitz.com/edit/stackblitz-starters-gzpusc?file=src%2Fstyle.css

**Comment by samhirtarif** [comment, marked as answer]
Since you are maintaining the selected value in the `Calendar` component, you could use `tileClassName` for that specific tile like [this](https://stackblitz.com/edit/stackblitz-starters-nvy34p?file=src%2FApp.tsx%3AL12).

[Here's](https://stackblitz.com/edit/stackblitz-starters-s7gauk?file=src%2FCalendar.css,src%2FCalendar.js) your stackblitz modified to cater to your requirement

**Reply by xsrokhan** [reply]
Thank you, your suggestion almost solves my issue, but it doesn't work for the current day. When you pull up the calendar, the current day has the purple background applied to it, but the number is visible only on hover, and it becomes invisible when you click away. Is there a solution to that? 
EDIT: Never mind, I found the solution to that, thanks again!
